test/utils/feature.flag.spec.ts

```diff
@@ -1,60 +1,32 @@
-import { describe, expect, test } from '@jest/globals';
+import { describe, expect } from '@jest/globals';
 import { isActiveFeature } from "../../src/utils/feature.flag";
 
-describe("feature flag tests", function() {
-
-  test("should return false if variable is 'false'", function() {
-    const active = isActiveFeature("false");
-    expect(active).toBeFalsy;
-  });
-
-  test("should return false if variable is '0'", function() {
-    const active = isActiveFeature("0");
+describe("feature flag tests", function () {
+  const dataExpectFalse = [
+    'false',
+    'faLse',
+    '0',
+    'off',
+    'oFF',
+    'kdjhskjf',
+  ];
+
+  it.only.each(dataExpectFalse)('should return false if variable is %p', (val) => {
```

Comment:
Nice one, but could you please use `test` (not its alias) and add `test` on the `import`

Reply:
changed to test, import not required

Reply:
Nope, the IDE will complain `Cannot find name 'test'. Do you need to install type definitions for a test runner? Try ...` but it is not a blocker